docker/dockerfiles/be-ubuntu.Dockerfile

```diff
@@ -0,0 +1,28 @@
+# This docker file build the Starrocks be ubuntu image
+# Please run this command from the git repo root directory to build:
+# DOCKER_BUILDKIT=1 docker build --rm=true -f docker/dockerfiles/be-ubuntu.Dockerfile -t starrocks/be-ubuntu:tag .
+
+ARG ARTIFACTIMAGE=artifact:latest
+FROM ${ARTIFACTIMAGE} as artifacts
+
+FROM ubuntu:22.04
+
+RUN apt-get update -y \
+        && apt-get install -y --no-install-recommends binutils-dev default-jdk python2 \
+           mysql-client curl vim tree net-tools \
+        && rm -rf /var/lib/apt/lists/*
+
+ENV JAVA_HOME=/lib/jvm/default-java
+
+ARG STARROCKS_ROOT=/opt/starrocks
```

Comment:
possibly add `ARG RUN_AS` or `ARG RUN_AS_ROOT`, we would prefer deliver non-root user `starrocks` as the default user to run in the container, and optionally allow overwrite it to `root` or other non-root username.

Reply:
will support non-root user after this PR is merged.